dashboard/test/fixtures/callout.yml

```diff
@@ -1,290 +1,459 @@
 callout_2:
   id: 2
-  element_id: '[block-id=''1'']'
+  element_id: "[block-id='1']"
   localization_key: drag_blocks
-  created_at: 2015-10-16 22:40:21.000000000 Z
-  updated_at: 2015-10-16 22:40:21.000000000 Z
+  created_at: !ruby/object:ActiveSupport::TimeWithZone
```

Comment:
This line makes me think that there is a problem independent of lesson groups. Please try removing any of the changes related to lesson groups, and get generate_fixtures.rb to the point where you can run it and get unit tests to pass on the fixtures it generates. then we can worry about adding lesson groups to the mix.